### PR TITLE
Cs/sc 2473 ga phone apk heartbeat ff

### PR DESCRIPTION
**Pull request by Charl1996**
## Product Description
This PR makes the `phone_apk_heartbeat` feature flag generally available from Standard Plan upwards.

## Technical Summary
[Ticket](https://dimagi-dev.atlassian.net/browse/SC-2473)

## Feature Flag
This change does not only apply to domains with a specific FF enabled, but it does apply to the `phone_apk_heartbeat` FF. 

### Safety story
Tested locally to make sure changes work

### Automated test coverage
No tests for this change. (there were none for the original feature and I don't think it's necessary to provide for this change also)

### Migrations
- [x] The migrations in this code can be safely applied first independently of the code

### Rollback instructions
If the toggle deletion needs to be reverted we need to identify domains that start using the feature without the toggle and then enable the toggle for those domains after reverting

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/accounting/migrations/0065_phone_apk_heartbeat_privs.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _grandfather_phone_apk_heartbeat_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        PHONE_APK_HEARTBEAT,
+        skip_edition='Community',
```
Comment (sravfeyn) [inline corehq/apps/accounting/migrations/0065_phone_apk_heartbeat_privs.py:16]: Shouldn't `Paused` be here as well?

**Review by sravfeyn — APPROVED** [review]

**Review by sravfeyn — CHANGES_REQUESTED** [review]
It looks like there are only four domains on Prod https://www.commcarehq.org/hq/flags/edit/show_build_profile_in_app_status/?show_service_type=true which all are on Internal, so we may not need to keep the toggle around at all, that is, we can remove the toggle altogether by including a migration that deletes the toggle.

**Comment by Charl1996** [comment]
@sravfeyn I think you viewed the wrong FF there. This correct one is the [phone_apk_heartbeat](https://www.commcarehq.org/hq/flags/edit/phone_apk_heartbeat/).

File: corehq/apps/accounting/migrations/0065_phone_apk_heartbeat_privs.py (hunk 1 above)
Comment (Charl1996) [reply corehq/apps/accounting/migrations/0065_phone_apk_heartbeat_privs.py:16]: @sravfeyn I'll make the update, but what is the `Paused` edition exactly?

**Review by SmittieC — COMMENTED** [review]
LGTM, but I'm going to let Sravan have the final say here

**Comment by sravfeyn** [comment]
Ah, you are right!

**Review by sravfeyn — APPROVED** [review]

File: corehq/apps/accounting/migrations/0065_phone_apk_heartbeat_privs.py (hunk 1 above)
Comment (sravfeyn) [reply corehq/apps/accounting/migrations/0065_phone_apk_heartbeat_privs.py]: I think this one https://confluence.dimagi.com/display/saas/CommCare+Subscriptions+-+Auto+Pause+Process https://confluence.dimagi.com/display/saas/CommCare+HQ+Accounting+-+Subscription+Creation+and+Modification

**Pull request merged**